### PR TITLE
Update UI for Handoff

### LINKED ISSUE #70: Consider adding new theme properties to editor
From [These Weeks in Firefox: Issue 32 ](https://blog.nightly.mozilla.org/2018/02/15/these-weeks-in-firefox-issue-32/):

> The WebExtension theme API now supports specifying colors for frame_inactive, tab_loading, icons and icons_attention, button_background, button_background_hover and button_background_active, toolbar_field_separator.

Need to play with these and see exactly what they do, filing this issue to not forget about it.

### PR DESCRIPTION
**Pull request by johngruen**
This technically closes #70 and implements a more scalable version of our UI

### CODE REVIEW

**Comment by pdehaan** [comment]
This is failing due to this rule in .stylelintrc:

https://github.com/mozilla/FirefoxColor/blob/c91296153392d387ea24d63f17da5262747ef717/.stylelintrc#L10

File: src/web/lib/components/App/index.js [hunk 1]
```diff
@@ -74,31 +76,56 @@ export const AppComponent = props => {
...
+    firstRun,
+    isFirefox,
+    addonUrl,
+    setSelectedColor,
+    selectedColor,
+    theme,
+    themeCustomImages
   } = props;
+
+  const customImages = (theme.images.custom_backgrounds || []).map(
```
Comment (johngruen) [inline src/web/lib/components/App/index.js]: This can be replaced with the themeHasCustomImages prop

Comment (johngruen) [reply src/web/lib/components/App/index.js]: eh, actually this way seems good

File: src/web/lib/components/AppHeader/index.scss [hunk 2]
```diff
@@ -1,15 +1,16 @@
-@import '../../common.scss';
+@import "../../common.scss";
 
 .app-header {
   align-items: center;
-  background: rgba(255, 255, 255, .9);
-  box-shadow: var(--small-unit-box-shadow);
+  background: var(--grey-10);
+  // box-shadow: var(--small-unit-box-shadow);
```
Comment (meandavejustice) [inline src/web/lib/components/AppHeader/index.scss]: nit: remove comment

**Comment by johngruen** [comment]
Aight @meandavejustice i think i got this fixed up

**Pull request merged**